packages/server/src/index.js

```diff
@@ -39,4 +40,14 @@ const cleanGeneratedPdfCron = new CronJob(
 );
 cleanGeneratedPdfCron.start();
 
+hasOutstandingMigrations().then((hasMigrations) => {
+    if (!hasMigrations) {
+        return;
+    }
+    console.error('There are outstanding db migrations. Run \'yarn db:migrate\' before trying again');
+    if (process.env.NODE_ENV === 'development') {
+        process.exit(1);
```

Comment:
Instead of crashing, would it be helpful to instead just run the migrations? I like the check, and it's definitely an improvement- just wondering if it could go a step further and eliminate the chore. 

Basically, I'm wondering if there's ever a scenario where this check crashes the app and a developer's immediate next step _wouldn't_ be running the migrations manually.